src/rules/no-unresolved.js

```diff
@@ -53,15 +53,17 @@ module.exports = function (context) {
     const modules = call.arguments[0]
     if (modules.type !== 'ArrayExpression') return
 
-    for (let element of modules.elements) {
-      if (element.type !== 'Literal') continue
-      if (typeof element.value !== 'string') continue
-
-      if (element.value === 'require' ||
-          element.value === 'exports') continue // magic modules: http://git.io/vByan
-
-      checkSourceValue(element)
-    }
+    modules.elements.forEach((element) => {
```

Comment:
I would prefer that these `continue`s become `returns`, to minimize deep nesting. but that's just code style preference, no impact on semantics/performance 🤓